tools/StandardAnchorTags/TocSectionNumberBuilder.cs

```diff
@@ -53,12 +53,24 @@ public TocSectionNumberBuilder(string pathFromToolToStandard, bool dryRun)
         /// <summary>
         /// Add the front matter entries to the TOC
         /// </summary>
-        /// <param name="entries">A params array of tuples that contains the link text and the file names.</param>
-        public void AddFrontMatterTocEntries(params (string linkText, string fileName)[] entries)
+        /// <param name="fileName">The front matter file name.</param>
+        /// <remarks>
+        /// The front matter entries don't add section numbers in the headers.
+        /// Otherwise, this is the same logic for the main file.
+        /// </remarks>
+        public async Task AddFrontMatterTocEntries(string fileName)
         {
-            foreach (var entry in entries)
+            string? line;
+            using var stream = new StreamReader($"{PathToStandardFiles}/{fileName}");
```

Comment:
Maybe use Path.Combine here? (I'm sure it's fine either way... just my habit.)
(I also generally use `File.OpenText` but it's a personal preference thing...)

Reply:
Good suggestion. I applied both.